docs/reference/commandline/tag.md

```diff
@@ -35,7 +35,7 @@ components may contain lowercase letters, digits and separators. A separator
 is defined as a period, one or two underscores, or one or more dashes. A name
 component may not start or end with a separator.
 
-A tag name must be valid ASCII and may contain lowercase and uppercase letters,
+A tag name must be valid ASCII and may contain lowercase letters,
```

Comment:
I think I misunderstood the error message. It is indeed the repository that is uppercase, not the tag.

$ docker tag 66f8366f51cc mattray/R:testing
Error parsing reference: "mattray/R:testing" is not a valid repository/tag
$ docker tag 66f8366f51cc mattray/r:testing
$ docker version
Client:
 Version:      17.03.1-ce
 API version:  1.27
 Go version:   go1.7.5
 Git commit:   c6d412e
 Built:        Tue Mar 28 00:40:02 2017
 OS/Arch:      darwin/amd64